appinventor/appengine/war/Ya.css

```diff
@@ -1030,6 +1030,11 @@ select.ode-PropertyEditor[disabled] {
   color: #8fc202;
 }
 
+.ode-ComponentHelpPopup > .popupContent > table{
+	width: 100%;
```

Comment:
please watch out for indentation. Should be 2 spaces (4 for continuation lines). Also note a space before the '{' in other css rules in this file.